### ISSUE TITLE
I can't figure out how to run my analyzer.

### ISSUE THREAD

**Issue by chlenc** [camden-smallwood/sway-analyzer#2]
# I can't figure out how to run my analyzer.
Hi guys, I stumbled upon my analyzer, which is very necessary for the development tool
I tried to run it, but nothing works, please help me.

I have a project with 7 linked smart contracts and I would like to analyze reentrancy problems using a sway analyzer.

I just installed it using the command
```
cargo install sway-analyzer
```

and after that, I went to the main project folder (where I have cargo.toml for tests) and started the analyzer with the command ``way-analyzer``, but nothing happened.

After that, I went to one of the contracts and ran it there, but there was no output either after that, I tried in the src folder, but no results either

## My toolchain
```
Default host: aarch64-apple-darwin
fuelup home: /Users/alexey/.fuelup

installed toolchains
--------------------
beta-3-aarch64-apple-darwin
beta-4-rc.2-aarch64-apple-darwin
latest-aarch64-apple-darwin (default)
beta-4-aarch64-apple-darwin
hotfix
latest-2023-09-25-aarch64-apple-darwin
my-custom-toolchain

active toolchain
-----------------
latest-aarch64-apple-darwin (default)
  forc : 0.46.0
    - forc-client
      - forc-deploy : 0.46.0
      - forc-run : 0.46.0
    - forc-doc : 0.46.0
    - forc-explore : 0.28.1
    - forc-fmt : 0.46.0
    - forc-index : 0.21.1
    - forc-lsp : 0.46.0
    - forc-tx : 0.46.0
    - forc-wallet : 0.3.0
  fuel-core : 0.20.7
  fuel-core-keygen : Error getting version string
  fuel-indexer : 0.21.1

fuels versions
---------------
forc : 0.45
forc-wallet : 0.45
```

## Sway analyzer version
`sway-analyzer 0.1.5`

**Comment by camden-smallwood** [comment]
Hi there, you need to supply either the directory or files that you would like to analyze.

In order to use the directory method, change into the folder that contains `Forc.toml` and run the following:
```
sway-analyzer --directory .
```

If you would like to supply specific files, run the following instead:
```
sway-analyzer --files ./src/main.sw
```

Hope that works for you!

**Comment by camden-smallwood** [comment]
The tool now displays the help message when no options are provided as of commit [22c6d08b96ec330edc07f8b091d2d532c2a27833](https://github.com/camden-smallwood/sway-analyzer/commit/22c6d08b96ec330edc07f8b091d2d532c2a27833), which has been released as version `0.1.6`.
